licensing the repository (#174)
bump-sh/docs: README.md

* Ordered lists: if you need to any kind of blocks in your ordered list item, make sure it immediately follows the numbered paragraph line (no empty line in between). You might also want to [indent your block to line up with the first non-space character after the list item marker](https://stackoverflow.com/questions/34987908/embed-a-code-block-in-a-list-item-with-proper-indentation-in-kramdown). Just do both for safety!

Then you can let your ~~pen~~ keyboard handle the content!

## License

> This repository has two main content:

Reviewer: ```suggestion
This repository has two main parts:
```
"Content" is uncountable.

> - The codebase making the website display correctly in your browser,
>   which is built on top of the Bridgetown framework. **Those files are
>   [MIT licensed](./LICENSE_MIT)**.

Reviewer: ```suggestion
- The codebase that makes the website display correctly in your browser
  is built on top of the Bridgetown framework. **Those files are
  [MIT licensed](./LICENSE_MIT)**.
```
